All Stia Side Task Dialogue

allister-translator opened the pull request:
take your time proofreading; im probably only like halfway done

allister-translator committed:
Update and rename Files/ssn_015301.csv to Translated/Dialog/Kirishima/ssn_015301.csv

allister-translator committed:
Update and rename Files/ssq_015302.csv to Translated/Dialog/Kirishima/ssq_015302.csv

allister-translator committed:
Update and rename Files/ssq_015304.csv to Translated/Dialog/Kirishima/ssq_015304.csv

allister-translator committed:
Update and rename Files/ssq_015305.csv to Translated/Dialog/Kirishima/ssq_015305.csv

allister-translator committed:
Update and rename Files/ssq_015307.csv to Translated/Dialog/Chitose/ssq_015307.csv

allister-translator committed:
Update and rename Files/ssq_015308.csv to Translated/Dialog/Kirishima/ssq_015308.csv

allister-translator committed:
Update and rename Files/ssn_015300.csv to Translated/Dialog/Kirishima/ssn_015300.csv

allister-translator committed:
Update and rename Files/ssn_015303.csv to Translated/Dialog/Kirishima/ssn_015303.csv

allister-translator committed:
Update ssn_015303.csv

allister-translator committed:
Update and rename Files/ssn_015306.csv to Translated/Dialog/Chitose/ssn_015306.csv

allister-translator committed:
Update ssn_015306.csv

allister-translator committed:
Update and rename Files/ssn_015331.csv to Translated/Dialog/Katsuragi/ssn_015331.csv

allister-translator committed:
Update and rename Files/ssn_015190_007.csv to Translated/Dialog/Rediah/ssn_015190_007.csv

allister-translator committed:
Merge branch 'Arks-Layer:EN_Reboot' into EN_Reboot

allister-translator committed:
Update and rename Files/ssn_015190_009.csv to Translated/Dialog/Rediah/ssn_015190_009.csv

allister-translator committed:
Update and rename Files/ssn_015190_013.csv to Translated/Dialog/Rediah/ssn_015190_013.csv

allister-translator committed:
Update and rename Files/ssn_015190_015.csv to Translated/Dialog/Rediah/ssn_015190_015.csv

allister-translator committed:
Update and rename Files/ssq_015190_008.csv to Translated/Dialog/Rediah/ssq_015190_008.csv

allister-translator committed:
Update and rename Files/ssq_015190_010.csv to Translated/Dialog/Rediah/ssq_015190_010.csv

allister-translator committed:
Update and rename Files/ssn_015190_011.csv to Translated/Dialog/Rediah/ssn_015190_011.csv

allister-translator committed:
Update ssn_015190_011.csv

allister-translator committed:
Update and rename Files/ssq_015190_012.csv to Translated/Dialog/Rediah/ssq_015190_012.csv

allister-translator committed:
Update and rename Files/ssq_015190_014.csv to Translated/Dialog/Rediah/ssq_015190_014.csv

allister-translator committed:
Update and rename Files/ssq_015190_016.csv to Translated/Dialog/Rediah/ssq_015190_016.csv

allister-translator committed:
Update and rename Files/ssn_015350.csv to Translated/Dialog/Yunoka/ssn_015350.csv

allister-translator committed:
Merge branch 'Arks-Layer:EN_Reboot' into EN_Reboot

allister-translator committed:
Update and rename Files/ssn_015351.csv to Translated/Dialog/Yunoka/ssn_015351.csv

allister-translator committed:
Update and rename Files/ssn_015354.csv to Translated/Dialog/Yunoka/ssn_015354.csv

allister-translator committed:
Update and rename Files/ssn_015357.csv to Translated/Dialog/Yunoka/ssn_015357.csv

allister-translator committed:
Update and rename Files/ssq_015352.csv to Translated/Dialog/Yunoka/ssq_015352.csv

allister-translator committed:
Update and rename Files/ssq_015353.csv to Translated/Dialog/Yunoka/ssq_015353.csv

allister-translator committed:
Update ssq_015353.csv

allister-translator committed:
Update and rename Files/ssq_015355.csv to Translated/Dialog/Yunoka/ssq_015355.csv

allister-translator committed:
Update and rename Files/ssq_015356.csv to Translated/Dialog/Yunoka/ssq_015356.csv

allister-translator committed:
Update and rename Files/ssq_015358.csv to Translated/Dialog/Yunoka/ssq_015358.csv

allister-translator committed:
Update and rename Files/ssq_015359.csv to Translated/Dialog/Yunoka/ssq_015359.csv

allister-translator committed:
Update and rename Files/ssn_015630.csv to Translated/Dialog/Souya/ssn_015630.csv

allister-translator committed:
Update ssn_015630.csv

allister-translator committed:
Update and rename Files/ssn_015631.csv to Translated/Dialog/Souya/ssn_015631.csv

allister-translator committed:
Update ssn_015630.csv

allister-translator committed:
Update and rename Files/ssn_015340.csv to Translated/Dialog/Marimo/ssn_015340.csv

allister-translator committed:
Update and rename Files/ssn_015341.csv to Translated/Dialog/Marimo/ssn_015341.csv

allister-translator committed:
Update and rename Files/ssq_015342.csv to Translated/Dialog/Marimo/ssq_015342.csv

allister-translator committed:
Update ssn_015357.csv

allister-translator committed:
Update ssn_015357.csv

allister-translator committed:
i just went with gigantz cuz i like Gantz

allister-translator committed:
Update and rename Files/ssq_015332.csv to Translated/Dialog/Katsuragi/ssq_015332.csv

allister-translator committed:
Update and rename Files/ssn_015330.csv to Translated/Dialog/Katsuragi/ssn_015330.csv

allister-translator committed:
Update ssn_015330.csv

allister-translator committed:
Update ssn_015334.csv

allister-translator committed:
Update and rename Files/ssn_015311.csv to Translated/Dialog/Souya/ssn_015311.csv

allister-translator committed:
Update ssn_015311.csv

allister-translator committed:
Update and rename Files/ssn_015320.csv to Translated/Dialog/Akatsuki/ssn_015320.csv

allister-translator committed:
Update and rename Files/ssn_015321.csv to Translated/Dialog/Akatsuki/ssn_015321.csv

allister-translator committed:
Update ssn_015320.csv

allister-translator committed:
Update and rename Files/ssq_015322.csv to Translated/Dialog/Akatsuki/ssq_015322.csv

allister-translator committed:
Update and rename Files/ssq_015335.csv to Translated/Dialog/Katsuragi/ssq_015335.csv

allister-translator committed:
Merge branch 'Arks-Layer:EN_Reboot' into EN_Reboot

allister-translator committed:
Update and rename Files/ssn_015310.csv to Translated/Dialog/Souya/ssn_015310.csv

allister-translator committed:
Update ssn_015310.csv

allister-translator committed:
Update ssn_015310.csv

allister-translator committed:
Update and rename Files/ssq_015312.csv to Translated/Dialog/Souya/ssq_015312.csv

allister-translator committed:
Update and rename Files/ssq_015313.csv to Translated/Dialog/Souya/ssq_015313.csv

allister-translator committed:
Update and rename Files/ssq_015513.csv to Translated/Dialog/Shioji/ssq_015513.csv

allister-translator committed:
Update ssq_015513.csv

allister-translator committed:
Merge branch 'Arks-Layer:EN_Reboot' into EN_Reboot

DabirSA committed:
Update ssn_015320.csv

DabirSA committed:
Update ssn_015330.csv

allister-translator committed:
Update and rename Files/ssn_015451.csv to Translated/Dialog/Souya/ssn_015451.csv

DabirSA committed:
Update ssn_015331.csv

allister-translator committed:
Update and rename Files/ssn_015461.csv to Translated/Dialog/Marimo/ssn_015461.csv

allister-translator committed:
Update and rename Files/ssn_015463.csv to Translated/Dialog/Marimo/ssn_015463.csv

DabirSA committed:
Update ssn_015331.csv

DabirSA committed:
Update ssn_015334.csv

DabirSA committed:
Update ssn_015303.csv

DabirSA committed:
Update ssq_015308.csv

DabirSA committed:
Update ssn_015300.csv

DabirSA committed:
Update ssn_015310.csv

DabirSA committed:
Update ssn_015311.csv

DabirSA committed:
Update ssq_015312.csv

DabirSA committed:
Update ssq_015313.csv

DabirSA committed:
Update ssn_015350.csv

allister-translator committed:
Update and rename Files/ssn_015471.csv to Translated/Dialog/Chitose/ssn_015471.csv

allister-translator committed:
Update and rename Files/ssn_015470.csv to Translated/Dialog/Chitose/ssn_015470.csv

allister-translator committed:
Update and rename Files/ssq_015452.csv to Translated/Dialog/Souya/ssq_015452.csv

allister-translator committed:
Update and rename Files/ssq_015472.csv to Translated/Dialog/Chitose/ssq_015472.csv

allister-translator committed:
Update and rename Files/ssq_015464.csv to Translated/Dialog/Marimo/ssq_015464.csv

allister-translator committed:
Update and rename Files/ssq_015462.csv to Translated/Dialog/Marimo/ssq_015462.csv

HaiMisterGuy commented on Translated/Dialog/Katsuragi/ssn_015330.csv:
```diff
@@ -0,0 +1,20 @@
+ssn_015330_1102#0,"""It's just... This time might be a little dangerous, so I'm not saying you have to."""
+ssn_015330_1201#0,"""Please collect data on the Recon Gigantix. All I need is for you to get close and confirm a visual."""
+ssn_015330_1202#0,"""The targets' strengths are unknown. Please take utmost care out there."""
+ssn_015330_1301#0,"""I seee... Well, if you change your mind, please let me know."""
```
'I seee' a typo

allister-translator replied:
fixt

HaiMisterGuy commented on Translated/Dialog/Katsuragi/ssn_015334.csv:
```diff
@@ -0,0 +1,13 @@
+ssn_015334_0010#0,"""You will?! Oh, this helps so much. Alright, please let me explain."""
+ssn_015334_0020#0,"""The other day, while in the Dreisan Plant, a scouting party encountered a certain group of DOLLS."""
+ssn_015334_0030#0,"""It seems that, compared to the average DOLLS, they had a noticeably different, heavy presence to them..."""
+ssn_015334_0040#0,"""They had no sooner engaged the DOLLS than they found themselves overpowered, with no choice but to retreat."""
```
than > that?

allister-translator replied:
they had no sooner (done this) than (that) 
it checks out grammatically, but i could probably reword it. 
i do see a misspelled Dreisen Plant tho 👀

allister-translator committed:
Update and rename Files/ssn_015481.csv to Translated/Dialog/Shirayuki/ssn_015481.csv

allister-translator committed:
Update ssn_015190_015.csv

allister-translator committed:
Update and rename Files/ssn_015480.csv to Translated/Dialog/Shirayuki/ssn_015480.csv

allister-translator committed:
Update and rename Files/ssn_015483.csv to Translated/Dialog/Shirayuki/ssn_015483.csv

HaiMisterGuy commented on Translated/Dialog/Kirishima/ssn_015303.csv:
```diff
@@ -0,0 +1,11 @@
+ssn_015303_0050#0,"""So, I worry for her safety..."""
+ssn_015303_0060#0,"""We were able to pick up her vitals, and can confirm she's uninjured. However, due to some malfunction with her Mag, we can't contact her."""
+ssn_015303_0070#0,"""Other troops are currently en route to rescue her. ...Alas, they seem to be short-staffed."""
+ssn_015303_0080#0,"""So I'd like you to enlist your help. Please see that she comes home safely."""
```
I'd like to enlist your help

allister-translator committed:
Update and rename Files/ssn_015450.csv to Translated/Dialog/Souya/ssn_015450.csv

HaiMisterGuy commented on Translated/Dialog/Marimo/ssn_015340.csv:
```diff
@@ -0,0 +1,13 @@
+name_7443#0,"""Marimo"""
+ssn_015340_0001#0,"""You must be... one of the recent arrivals in Stia, yes?"""
+ssn_015340_0002#0,"""Have you been holding up well against the heat? Please remember to hydrate yourself before battle."""
+ssn_015340_0003#0,"""If you were to collapse before your enemy did, that would defeat the whole purpose."""
```
"that would defeat the whole purpose" doesn't make sense for me given the context. Is it suppose to suggest that it defeats the purpose of coming to Stia?

allister-translator committed:
Update and rename Files/ssn_015460.csv to Translated/Dialog/Marimo/ssn_015460.csv

HaiMisterGuy commented on Translated/Dialog/Shirayuki/ssn_015481.csv:
```diff
@@ -0,0 +1,9 @@
+name_7441#0,"""Shirayuki"""
+ssn_015481_0010#0,"""Even within my team, word of your feats is becoming a common topic."""
+ssn_015481_0020#0,"""Some of the troops would also like to study how you fight."""
+ssn_015481_0030#0,"""If it's alright with you, I want you to let them watch you fight enemies."""
```
Repetition of 'you', may read better saying

> I'd like them watch you fight enemies

allister-translator committed:
Update and rename Files/ssn_015491.csv to Translated/Dialog/Hatsukari/ssn_015491.csv

HaiMisterGuy commented on Translated/Dialog/Souya/ssn_015310.csv:
```diff
@@ -0,0 +1,11 @@
+name_7446#0,"""Souya"""
+ssn_015310_0001#0,"""In a protracted war scenario, supply replenishment, as well as management of said, is more important than anything else."""
```
May read better saying

> supplying replenishments, as well as management of so

If you rather keep the 'said' wording, then perhaps

> as well as management of said items

allister-translator committed:
Update and rename Files/ssq_015482.csv to Translated/Dialog/Shirayuki/ssq_015482.csv

allister-translator committed:
Update and rename Files/ssq_015484.csv to Translated/Dialog/Shirayuki/ssq_015484.csv

allister-translator committed:
Update and rename Files/ssq_015492.csv to Translated/Dialog/Hatsukari/ssq_015492.csv

allister-translator committed:
Update ssn_015491.csv

allister-translator committed:
Update and rename Files/ssn_015490.csv to Translated/Dialog/Hatsukari/ssn_015490.csv

allister-translator committed:
Update ssn_015490.csv

allister-translator committed:
Update ssn_015460.csv

allister-translator committed:
Update ssn_015330.csv

allister-translator committed:
dreisen + rewording

allister-translator committed:
Update ssn_015303.csv

allister-translator committed:
Update ssn_015481.csv

allister-translator committed:
dunno how to reword it so i just changed the line

allister-translator committed:
Update ssn_015310.csv

HaiMisterGuy commented on Translated/Dialog/Hatsukari/ssn_015490.csv:
```diff
@@ -0,0 +1,12 @@
+ssn_015490_0002#0,"""The call from Central was rather sudden. Sorry for the lack of a more official welcome!"""
+ssn_015490_0003#0,"""I can't speak for everyone else, but I'm glad to have ya with us!"""
+ssn_015490_0101#0,"""Yo! The name's Hatsukari. I'm the Commander of the 2nd Combat Platoon here in Stia."""
+ssn_015490_0102#0,"""For someone who ain't used to this place, but you do good work. Honestly, it'll help me out as well."""
```
-but

allister-translator committed:
Update ssn_015350.csv

allister-translator committed:
Merge branch 'Arks-Layer:EN_Reboot' into EN_Reboot

allister-translator committed:
Update ssn_015490.csv

allister-translator committed:
Update and rename Files/ssn_015600.csv to Translated/Dialog/Akatsuki/ssn_015600.csv

allister-translator committed:
Update and rename Files/ssn_015601.csv to Translated/Dialog/Akatsuki/ssn_015601.csv

allister-translator committed:
Update and rename Files/ssq_015602.csv to Translated/Dialog/Akatsuki/ssq_015602.csv

allister-translator committed:
Update ssn_015600.csv

allister-translator committed:
Update and rename Files/ssn_015603.csv to Translated/Dialog/Akastuki/ssn_015603.csv

allister-translator committed:
Update and rename Files/ssq_015604.csv to Translated/Dialog/Akatsuki/ssq_015604.csv

allister-translator committed:
Update and rename Files/ssn_015605.csv to Translated/Dialog/Akatsuki/ssn_015605.csv

allister-translator committed:
Update and rename Files/ssq_015606.csv to Translated/Dialog/Akatsuki/ssq_015606.csv

allister-translator committed:
Update and rename Files/ssn_015500.csv to Translated/Dialog/Anan/ssn_015500.csv

allister-translator committed:
Update and rename Files/ssn_015501.csv to Translated/Dialog/Anan/ssn_015501.csv

allister-translator committed:
Update ssn_015501.csv

allister-translator committed:
Update ssn_015501.csv

allister-translator committed:
Update ssn_015500.csv

allister-translator committed:
Update and rename Files/ssn_015502.csv to Translated/Dialog/Anan/ssn_015502.csv

allister-translator committed:
Update and rename Files/ssn_015510.csv to Translated/Dialog/Shioji/ssn_015510.csv

allister-translator committed:
Update and rename Files/ssn_015511.csv to Translated/Dialog/Shioji/ssn_015511.csv

allister-translator committed:
Update and rename Files/ssn_015514.csv to Translated/Dialog/Shioji/ssn_015514.csv

allister-translator committed:
Update ssn_015500.csv

allister-translator committed:
Update and rename Files/ssn_015610.csv to Translated/Dialog/Hatsukari/ssn_015610.csv

allister-translator committed:
Update and rename Files/ssn_015611.csv to Translated/Dialog/Hatsukari/ssn_015611.csv

allister-translator committed:
Update and rename Files/ssq_015612.csv to Translated/Dialog/Hatsukari/ssq_015612.csv

allister-translator committed:
Update and rename Files/ssn_015620.csv to Translated/Dialog/Marimo/ssn_015620.csv

allister-translator committed:
Update and rename Files/ssn_015621.csv to Translated/Dialog/Marimo/ssn_015621.csv

allister-translator committed:
Update and rename Files/ssq_015622.csv to Translated/Dialog/Marimo/ssq_015622.csv

allister-translator committed:
Update and rename Files/ssn_015640.csv to Translated/Dialog/Shirayuki/ssn_015640.csv

allister-translator committed:
Update ssn_015640.csv

allister-translator committed:
Update and rename Files/ssn_015641.csv to Translated/Dialog/Shirayuki/ssn_015641.csv

DabirSA committed:
Update ssn_015603.csv

DabirSA committed:
Update ssn_015605.csv

DabirSA committed:
Update ssn_015600.csv

DabirSA committed:
Update ssn_015490.csv

DabirSA committed:
Update ssn_015611.csv

allister-translator committed:
Update and rename Files/ssn_015651.csv to Translated/Dialog/Hokuto/ssn_015651.csv

DabirSA committed:
Update ssn_015651.csv

DabirSA committed:
Update ssn_015334.csv

allister-translator committed:
Update and rename Files/ssn_015652.csv to Translated/Dialog/Hokuto/ssn_015652.csv

DabirSA committed:
Update ssn_015621.csv

DabirSA committed:
Update ssn_015610.csv

DabirSA committed:
Update ssn_015500.csv

DabirSA committed:
Update ssn_015510.csv

DabirSA commented on Translated/Dialog/Shioji/ssn_015510.csv:
```diff
@@ -0,0 +1,39 @@
+ssn_015510_1403#0,"""It's strong, and yet it'd be easy to process."""
+ssn_015510_1404#0,"""...Ahh, just looking at it, all sorts of possibilities come to mind."""
+ssn_015510_1405#0,"""We could make some incredible things with this, there's no doubt about that."""
+ssn_015510_1406#0,"""It's been a while since the researcher in me was this fired up! I look forward to the coming days!"""
```
This last bit could do with rewording.

allister-translator replied:
get ur head out of the gutter

DabirSA replied:
Didn't even occur to me. I just think it reads a bit unnatural

allister-translator committed:
Update and rename Files/ssn_015650.csv to Translated/Dialog/Hokuto/ssn_015650.csv

allister-translator committed:
Update and rename Files/ssq_015512.csv to Translated/Dialog/Shioji/ssq_015512.csv

allister-translator committed:
Update and rename Files/ssq_015515.csv to Translated/Dialog/Shioji/ssq_015515.csv

DabirSA committed:
Update ssq_015512.csv

allister-translator committed:
Update and rename Files/ssq_015516.csv to Translated/Dialog/Shioji/ssq_015516.csv

DabirSA committed:
Update ssq_015515.csv

DabirSA committed:
Update ssq_015516.csv

allister-translator committed:
Update and rename Files/ssn_015750.csv to Translated/Dialog/Hatsukari/ssn_015750.csv

DabirSA committed:
Update ssn_015650.csv

allister-translator committed:
creative license

he doesn't actually say this and i swapped the original positions of two lines, but it's close enough and it flows better
